packages/react-router/test-app/package.json

```diff
@@ -15,7 +15,7 @@
     "@types/react-dom": "^16.9.4",
     "@types/react-router": "^5.1.8",
     "@types/react-router-dom": "^5.1.7",
-    "ionicons": "^5.4.0",
+    "ionicons": "^5.5.4",
```

Comment:
Can get a tech debt ticket for this? The test app should probably be on v6 for everything.

Reply:
We aren't affected by the Ionicons v6 breaking changes, so I was able to update this in: https://github.com/ionic-team/ionic-framework/pull/26265/commits/84cbcae0b694e24751f66f3fbdc5da182fe4b0cf